code/go/0chain.net/smartcontract/storagesc/handler.go

```
@@ -254,23 +256,20 @@ func (ssc *StorageSmartContract) GetAllocationMinLockHandler(ctx context.Context
 	allocData := params.Get("allocation_data")
 	var request newAllocationRequest
 	if err = request.decode([]byte(allocData)); err != nil {
-		return "", common.NewErrInternal("can't decode allocation request", err.Error())
+		logging.Logger.Error("can't decode allocation request",
+			zap.Error(err))
```

Return error instead of logging only